api/server/structs/other.go

```diff
@@ -273,7 +273,7 @@ type PendingDeposit struct {
 }
 
 type PendingPartialWithdrawal struct {
-	Index             string `json:"index"`
+	ValidatorIndex    string `json:"index"`
```

Comment:
If the name here changed shouldn't the json too?
```suggestion
	ValidatorIndex    string `json:"validator_index"`
```